WISE-Community/WISE-Client
# refactor(Export): Label component export extends new abstract component

## Changes
- Refactored LabelComponentDataExportStrategy.ts to extend the new abstract component data export class
- Deleted the old AbstractComponentDataExportStrategy.ts
- Renamed ComponentDataExportStrategy.ts to AbstractComponentDataExportStrategy.ts

## Test
- Make sure Label export all and latest work
- Make sure Open Response export works
- Make sure Discussion export works
- Make sure Peer Chat export works
